src/howTos/dev/runConnector.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Standalone run a connector from an external app
+summary: How to run an monitor a connector standalone without any user setup
+---
+
+Running a connector is normally handled by the [Home application](https://github.com/cozy/cozy-home)
+with [cozy-harvest-lib](https://github.com/cozy/cozy-libs/tree/master/packages/cozy-harvest-lib).
+But if you are in an environment which cannot use react (for example in a native application), you will have to run your connector
+yourself.
+
+If you are in a Javascript environment, you can use the [cozy-client](/cozy-client/getting-started/) lib.
+Or else, you still have the [cozy-stack api](/cozy-stack/#reference).
+
+The whole process is detailed in the [Konnector workflow](/cozy-stack/konnectors-workflow/).
+you will get here a practical example.
+
+We will suppose that the cozy url is `https://runconnectordemo.mycozy.cloud` and that the
+connector slug is `connectorslug`.
+
+The login will be `userlogin` and the password will be `userpassword`
+
```

Comment:
I think we can have a small resume: 

1. Get an access token
2. Create an account 
3. Create a trigger 
4. Launch the trigger and get a job
5.  Follow the job execution